### ISSUE TITLE
Is user auth through v2.0 endpoint supported?

### ISSUE THREAD

**Issue by mitchhymel** [Microsoft/groove-api-sdk-csharp#5]
Hi,

I'm working on a desktop app that will integrate with Groove. I am currently trying to use [MSAL](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-v2-devquickstarts-wpf) for user authentication. The authentication process seems to go successfully, but when using the obtained token for Groove API calls, I see "INVALID_AUTHORIZATION_HEADER" error. 

Does this mean that I cannot use MSAL to obtain a user token that would work with Groove API? 

It's possible I am providing the wrong scope for the token request. However, if I provide the scope of "MicrosoftMediaService.GrooveApiAccess" to the PublicClientApplication when getting a token, the login pop up hits the following error: 
```
AADSTS70011: The provided value for the input parameter 'scope' is not valid. The scope MicrosoftMediaServices.GrooveApiAccess openid email profile offline_access is not valid.
```

Here is my code for obtaining the token from my implementation of IUserTokenManager, using PublicClientApplication from MSAL:

```
public async Task<string> GetUserAuthorizationHeaderAsync(bool forceRefresh = false)
{
    string clientId = "MyClientId"
    string[] scope = new string[]{ clientId }; // Is this the correct scope?
    PublicClientApplication app = new PublicClientApplication(clientId);
    AuthenticationResult result = await app.AcquireTokenAsync(scope); // result.Token looks like a valid token
    return (result != null && result.Token != null) ? $"Bearer {result.Token}": null;
}
```

**Comment by antoinecellerier** [comment]
Hi Mitch,

Azure active directory is not a supported authentication mechanism today. I would recommend that you use authentication flows documented at https://docs.microsoft.com/en-us/groove/using-the-groove-restful-services/user-authentication . If those don't work for you we'd love to understand why.

**Comment by mitchhymel** [comment]
Hi Antoine,

It's unfortunate that AAD is not supported currently. However, I was able to get user authentication working. The documentation you linked was helpful, but I also found [this](https://github.com/liveservices/LiveSDK-for-Windows/tree/master/src/Desktop/Samples/ApiExplorer) sample very helpful with regards to how to perform the authentication flow specifically for a desktop app.

Thanks for the help!
